embeddings/evaluate_sequence_tagging.py

```diff
@@ -0,0 +1,146 @@
+import argparse
+import json
+import os
+from typing import Dict, List
+
+import flair
+import torch
+from flair.datasets import ColumnCorpus
+from flair.embeddings import TransformerWordEmbeddings
+from flair.models import SequenceTagger
+from flair.trainers import ModelTrainer
+from sklearn.model_selection import train_test_split
+
+from embeddings import datasets
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        '-d',
+        '--device',
+        help='Device for Flair e.g. cpu',
+        type=str,
+        required=True,
+    )
+    parser.add_argument(
+        '-e',
+        '--embedding',
+        help='Huggingface embedding model name',
+        type=str,
+        required=True,
+    )
+    parser.add_argument(
+        '-u',
+        '--url',
+        help='Dataset url',
+        type=str,
+        required=True,
+    )
+    parser.add_argument(
+        '-r',
+        '--root',
+        help='Root directory for output files',
+        type=str,
+        required=False,
+        default='resources/tagging/'
+    )
+    return parser.parse_args()
+
+
+def setup(device):
+    flair.device = torch.device(device)
+    flair.set_seed(441)
+
+
+def read_jsonl(path: str) -> List[Dict]:
+    with open(path, 'r', encoding='utf-8') as f:
+        data = f.readlines()
+        data = [json.loads(it) for it in data]
+    return data
+
+
+def split_train_test_dev(data):
+    train, test = train_test_split(data, test_size=0.4, random_state=1)
+    dev, test = train_test_split(test, test_size=0.5, random_state=1)
+    return {
+        'train': train,
+        'dev': dev,
+        'test': test
+    }
+
+
+def convert_jsonl_to_connl(data: List[Dict], out_path: str):
```

Comment:
separate it to `converters` or similar subpackage

Reply:
done, but still it need some refactoring to handle different input_data parsing scheme